Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonManager.cs

```diff
@@ -428,14 +431,20 @@ public static IPalette CurrentGlobalPalette
                         return PaletteOffice2007White;
                     case PaletteModeManager.Office2007Black:
                         return PaletteOffice2007Black;
+                    case PaletteModeManager.Office2007BlackAlt:
```

Comment:
Can this be a switch expression,
Or even a static Dictionary Lookup, so it can be reused.

Reply:
I'm going to change this to Default, Dark Mode & Light Mode